### PR TITLE
Solana Daily Stablecoin Balances

### PR DESCRIPTION
**Pull request by Synthquest**
## Thank you for contributing to Spellbook 🪄
Please open the PR in **draft** and mark as ready when you want to request a review. 

### Description:

[...]

---
quick links for more information:
- [README.md](https://github.com/duneanalytics/spellbook/blob/main/README.md)
- [spellbook docs](https://github.com/duneanalytics/spellbook/tree/main/docs)
- [CONTRIBUTING.md](https://github.com/duneanalytics/spellbook/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Comment by Synthquest** [comment]
@Hosuke 
Any idea why the schema isnt getting picked up?

it almost seems like the schema isnt getting configured right when it runs the test

**Comment by Synthquest** [comment]
@jeff-dude any ideas here? I'm not sure why the schema isn't getting picked up

**Comment by jeff-dude** [comment]
> @jeff-dude any ideas here? I'm not sure why the schema isn't getting picked up

i'm fairly certain it's due to where you placed the source. if you look at dbt project file, it expects to be in root directory sources:
https://github.com/duneanalytics/spellbook/blob/main/dbt_subprojects/solana/dbt_project.yml#L23

since you have in subproject sources, it can't find it?

**Comment by Synthquest** [comment]
> > @jeff-dude any ideas here? I'm not sure why the schema isn't getting picked up
> 
> i'm fairly certain it's due to where you placed the source. if you look at dbt project file, it expects to be in root directory sources: https://github.com/duneanalytics/spellbook/blob/main/dbt_subprojects/solana/dbt_project.yml#L23
> 
> since you have in subproject sources, it can't find it?

Still getting issues with "schema does not exist" on the run after moving sources to the root model . If you create a new folder do you have to define it anywhere for solana? I don't think I've had to do that with the other envs

**Comment by jeff-dude** [comment]
> Still getting issues with "schema does not exist" on the run after moving sources to the root model . If you create a new folder do you have to define it anywhere for solana? I don't think I've had to do that with the other envs

by root, i meant out of the subproject altogether. this source should live here:
https://github.com/duneanalytics/spellbook/blob/main/sources/_subprojects_outputs/solana/_sources.yml

File: dbt_subprojects/solana/models/_sector/stablecoins/stablecoins_solana_balances.sql [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+, updated_balances as (
+            select 
+                  day
+                  , bal.token_balance_owner
+                  , sd.symbol
+                  , sum(coalesce(token_balance,0)) as token_balance 
+                  , token_mint_address
+            from {{ ref('solana_utils_daily_balances') }}
+            inner join (select address, symbol from stable_definitions) sd on sd.address = bal.token_mint_address
```
Comment (jeff-dude) [inline dbt_subprojects/solana/models/_sector/stablecoins/stablecoins_solana_balances.sql:31]:
```suggestion
            inner join (select address, symbol from stablecoin_tokens) sd on sd.address = bal.token_mint_address
```

**Comment by Synthquest** [comment]
> > Still getting issues with "schema does not exist" on the run after moving sources to the root model . If you create a new folder do you have to define it anywhere for solana? I don't think I've had to do that with the other envs
> 
> by root, i meant out of the subproject altogether. this source should live here: https://github.com/duneanalytics/spellbook/blob/main/sources/_subprojects_outputs/solana/_sources.yml

still no luck

File: dbt_subprojects/solana/models/_sector/stablecoins/stablecoins_solana_balances.sql (hunk 1 above)
Comment (jeff-dude) [reply dbt_subprojects/solana/models/_sector/stablecoins/stablecoins_solana_balances.sql:31]: ah, i think this is it, CTE name mismatch. i knew i had seen that error before, couldn't recall exactly what. need to document this!

**Review by jeff-dude — CHANGES_REQUESTED** [review]

**Pull request closed without merging**